.github/workflows/upload-contrib-stats.yml

```diff
@@ -22,6 +24,7 @@ jobs:
         env:
           ROCKSET_API_KEY: ${{ secrets.ROCKSET_API_KEY }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
         run: |
-          echo "Uploading external contribution stats for $(date -v-1d +%F)"
-          python3 -m tools.stats.upload_external_contrib_stats --startDate "$(date -v-1d +%F)"
+          echo "Uploading external contribution stats for" "$(date -d yesterday '+%Y-%m-%d')"
```

Comment:
Per our discussion, the self-hosted runner has access to some S3 buckets because of its IAM role.  You would need to add the new bucket into the list to grant the runner access.  There is no AWS key setup here in the workflow